Amethyst/default.amethyst

```diff
@@ -62,6 +62,10 @@
         "mod": "mod1",
         "key": "q"
     },
+    "focus-screen-5": {
+        "mod": "mod1",
+        "key": "g"
```

Comment:
Just assigned a random new key that I see has not been assigned to another action yet.